src/code/stores/graph-store.coffee

```diff
@@ -391,7 +391,8 @@ GraphStore  = Reflux.createStore
 
     if isDoubleClick
       @selectionManager.selectNodeForInspection(link.targetNode)
-      InspectorPanelStore.actions.openInspectorPanel('relations', {link: link})
+      if AppSettingsStore.store.settings.simulationType != 0
```

Comment:
These tests would be clearer if expressed in terms of a `DIAGRAM_ONLY` constant rather than 0.